bigbone/src/test/kotlin/social/bigbone/api/method/SuggestionMethodsTest.kt

```diff
@@ -0,0 +1,29 @@
+package social.bigbone.api.method
+
+import org.amshove.kluent.shouldBeEqualTo
+import org.junit.jupiter.api.Assertions
+import org.junit.jupiter.api.Test
+import social.bigbone.api.exception.BigBoneRequestException
+import social.bigbone.testtool.MockClient
+
+class SuggestionMethodsTest {
+
+    @Test
+    fun getSuggestions() {
+        val client = MockClient.mock("suggestions.json")
+        val suggestionMethods = SuggestionMethods(client)
+        val suggestions = suggestionMethods.getSuggestions(2).execute()
+        suggestions.size shouldBeEqualTo 2
```

Comment:
Optional improvement:

```suggestion
        suggestions shouldHaveSize 2
```

(Beware: needs import `import org.amshove.kluent.shouldHaveSize`)